packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx

```diff
@@ -153,12 +153,12 @@ const SSICredentialIssueRequestPage: React.FC = () => {
                         marginTop: isNarrowScreen ? 20 : '15%',
                         alignItems: 'center'
                     }}>
-                        <NonMobile>
+                        <NonMobileOS>
```

Comment:
I preferred the previous approach here to use a cutoff point for the width of the screen. we can decrease that. but that's my preference.

Reply:
Showing a deeplink on a desktop is pointless. The deeplink is not going to work on a desktop device and a QR code is not going to work on a mobile device unless you have two phones.
 So the OS type controls which one of the two is displayed.